ibm/mas_devops/roles/cp4d_db2wh/defaults/main.yml

```diff
@@ -21,15 +21,15 @@ db2wh_num_pods: "{{ lookup('env', 'DB2WH_NUM_PODS') | default(1, true) }}" # by
 
 # Configure meta storage for db2 (CP4D v3.5 and v4.0)
 db2wh_meta_storage_class: "{{ lookup('env', 'DB2WH_META_STORAGE_CLASS') }}"
-db2wh_meta_storage_size_gb: "{{ lookup('env', 'DB2WH_META_STORAGE_SIZE_GB') | default('20Gi', true) }}"
+db2wh_meta_storage_size_gb: "{{ lookup('env', 'DB2WH_META_STORAGE_SIZE_GB') | default('100Gi', true) }}"
```

Comment:
What are the reasons for increasing the storage for these 3?

Reply:
By previous experience, using only 20Gi for db2wh system storage is not enough to install IoT + Manage in the same db... pretty quickly we start getting disk space issue on db2